maquetado/scss/styles.scss

```diff
@@ -1,3 +1,32 @@
-.title {
-  color: red;
+.bold {
+    font-weight: bold;
+}
+
+.book-title {
+    font-size: 32px;
+    margin-bottom: 20px;
+    margin-top: 10px;
```

Comment:
Avoid margin-top and margin-left, they usually make responsiveness more complex. Use margin-bottom/margin-right in the previous sibling.

Besides, here your are using margin-top to separate an element from its parent. For that, always use `padding`. Margins are for separating siblings, e.g:
```
// container --> padding: 10px, to have 10px of distance to its children
//                  --> flex-direction: column, display: flex
<div class="container"> 
  <span class="first-child"/> // margin-bottom: 5px, to separate itself to the next sibling
  <span class="first-child"/> // margin-bottom: 5px, to separate itself to the next sibling
  <span class="first-child"/> // No margin here, so the distance to the parent remains 10px
</div>
```